pkg/apis/core/v1/conversion_test.go

```diff
@@ -657,6 +658,14 @@ func Test_v1_NodeSpec_to_core_NodeSpec(t *testing.T) {
 			PodCIDR:  "ace:cab:deca::/8",
 			PodCIDRs: []string{"ace:cab:deca::/8", "10.0.1.0/24"},
 		},
+		// list only - 4
+		{
```

Comment:
https://github.com/kubernetes/kubernetes/blob/e1aa8197eddd277fb82be0b8bbc2ba2b4ca67af7/staging/src/k8s.io/api/core/v1/types.go#L5797
Should it be deleted when only PodCIDRS and PodCIDR not set? I can't make up my mind!

Reply:
local test passed